unified_ecommerce/test_utils.py

```diff
@@ -118,10 +155,20 @@ def assert_json_equal(obj1, obj2, sort=False):  # noqa: FBT002
         obj1 (object): the first object
         obj2 (object): the second object
         sort (bool): If true, sort items which are iterable before comparing
+
+    Keyword Args:
+        ignore_timestamps: If true, ignore timestamps in the comparison
     """  # noqa: D401
     renderer = JSONRenderer()
     converted1 = json.loads(renderer.render(obj1))
     converted2 = json.loads(renderer.render(obj2))
+
+    if kwargs.get("ignore_timestamps"):
+        converted1.pop("created_on", None)
+        converted1.pop("updated_on", None)
+        converted2.pop("created_on", None)
+        converted2.pop("updated_on", None)
```

Comment:
I think mutating the object that's passed in is probably not a good idea because it makes an assert function a side-effecting function which it shouldn't be nor is it communicated in the signature or docstring. It also doesn't solve the problem for any nested objects that get serialized with timestamps.

I'd propose using this library I've looked at [this library](https://pypi.org/project/anys/) in the past, it allows things like this:

```python
from anys import ANY_DATETIME_STR

assert_json_equal({
    {
        **serializer.data,
        "created_on": ANY_DATETIME_STR,
        "updated_on": ANY_DATETIME_STR,
    },
    dj_serializer
)

```

We could even have a constant for that, or a function that you could just pass `serializer.data` into:

```python

ANY_TIMESTAMPS = {
    "created_on": ANY_DATETIME_STR,
    "updated_on": ANY_DATETIME_STR,
}

assert_json_equal({
    {
        **serializer.data,
        **ANY_TIMESTAMPS
    },
    dj_serializer
)
```